tinkerbell/stack/values.yaml

```diff
@@ -32,7 +32,15 @@ stack:
     roleBindingName: kube-vip-rolebinding
     # Customize the interface KubeVIP advertises on. When unset, KubeVIP will autodetect the
     # interface.
-    #interface: eth0
+    #interface: enp0s8
+  relay: # relay allows us to listen and respond to layer broadcast DHCP requests
+    name: dhcp-relay
+    enabled: true
+    # This image (ghcr.io/jacobweinstock/dhcrelay) is used because `dhcrelay` doesn't respect signals properly when run as PID 1.
+    image: ghcr.io/jacobweinstock/dhcrelay
+    maxHopCount: 10
+    # sourceInterface is the Host/Node interface to use for listening for DHCP broadcast packets.
+    sourceInterface: eno1 # TODO(jacobweinstock): investigate autodetecting a default
```

Comment:
I need to update the comment here because its not clear the need for this. This interface is used to create a macvlan interface off of, see the link below. Then the newly created macvlan interface moved into the dhcp relay pod's network namespace. 

https://github.com/tinkerbell/charts/blob/ac09ecc5c8ee13d80b6b3d148827985e2e8c21e6/tinkerbell/stack/templates/relay.yaml#L47